tool/grind.dart

```diff
@@ -44,3 +44,21 @@ build() {
 deploy() {
   run('surge', arguments: ['-d', 'which-film.info', '-p', 'build/web']);
 }
+
+@Task('Run in browser.')
+@Depends(build)
+browser() {
+  try {
+    run('hash', arguments: ['dartium']);
```

Comment:
It will throw if you don't have dartium available, and runs the codepath below that uses Google Chrome...